common/cache_policy/lru_policy.go

```diff
@@ -116,3 +129,21 @@ func getKeyPair(node *list.Element) KeyPair {
 	//uncast the keypair
 	return node.Value.(*list.Element).Value.(KeyPair)
 }
+
+// return true if no eviction happened/cache full, return false otherwise
+func (cache *LRUCache) evict() bool {
```

Comment:
I think we can think of a better name for this since the boolean return semantics is the opposite of what you'd expect

Reply:
yea I can change to true being a successful eviction and false being unsuccessful